src/main/java/com/iota/iri/conf/ProtocolConfig.java

```diff
@@ -7,34 +7,59 @@
 public interface ProtocolConfig extends Config {
 
     /**
-     * @return Descriptions#MWM
+     * Default Value: {@value BaseIotaConfig.Defaults#MWM}
+     *
+     * @return {@value ProtocolConfig.Descriptions#MWM}
      */
     int getMwm();
 
     /**
-     * @return Descriptions#TRANSACTION_PACKET_SIZE
+     * Default Value: {@value BaseIotaConfig.Defaults#PACKET_SIZE}
+     *
+     * @return {@value ProtocolConfig.Descriptions#TRANSACTION_PACKET_SIZE}
      */
     int getTransactionPacketSize();
 
     /**
-     * @return Descriptions#REQUEST_HASH_SIZE
+     * Default Value: {@value BaseIotaConfig.Defaults#REQ_HASH_SIZE}
```

Comment:
the default value name does not match the description name.
`REQ_HASH_SIZE` vs `REQUEST_HASH_SIZE`